persist/adapter.go

```diff
@@ -46,9 +46,15 @@ type Adapter interface {
 	// AddPolicy adds a policy rule to the storage.
 	// This is part of the Auto-Save feature.
 	AddPolicy(sec string, ptype string, rule []string) error
+	// AddPolicies adds policy rules to the storage.
+	// This is part of the Auto-Save feature.
+	AddPolicies(secs []string, ptypes []string, rules [][]string) []error
 	// RemovePolicy removes a policy rule from the storage.
 	// This is part of the Auto-Save feature.
 	RemovePolicy(sec string, ptype string, rule []string) error
+	// RemovePolicies removes policy rules from the storage.
+	// This is part of the Auto-Save feature.
+	RemovePolicies(secs []string, ptypes []string, rules [][]string) []error
```

Comment:
It's completely unnecessary.

Reply:
Make sure these two functions are optional for an adapter.

Reply:
Okay will make changes.